.editorconfig

```diff
@@ -5,7 +5,7 @@ root = true
 charset = utf-8
 indent_style = space
 indent_size = 4
-insert_final_newline = false
```

Comment:
This shouldn't have been `false` in first place :(

Reply:
This was brought in from a haskell project. I think stylish-haskell dictates that the last line shouldn't have a newline